plugins/woocommerce-admin/client/activity-panel/activity-panel.js

```diff
@@ -73,6 +73,19 @@ export const ActivityPanel = ( { isEmbedded, query } ) => {
 	const { updateUserPreferences, ...userData } = useUserPreferences();
 	const activeSetupList = useActiveSetupTasklist();
 
+	const closePanel = () => {
```

Comment:
Was this causing an error, or just a "declare the function before you use it" kind of change?